<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureReportDialog</class>
 <widget class="QDialog" name="ConfigureReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Reporting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="17,20">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Triggers</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="4" column="1">
          <widget class="QSpinBox" name="customPeriod">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> day</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="period">
           <item>
            <property name="text">
             <string>Daily</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Weekly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Monthly</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="name">
           <property name="text">
            <string>Report</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>How often to send the report</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Period</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QRadioButton" name="summaryData">
           <property name="text">
            <string>Summary Data</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter a unique name for this report</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QPushButton" name="sendNow">
           <property name="text">
            <string>Send Report Now</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="useCustomPeriod">
           <property name="text">
            <string>Custom Period</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QRadioButton" name="allData">
           <property name="text">
            <string>All Data</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Summary sends just the min/max for each sensor. All sends all the measurements from the selected period</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="sensorGroup">
        <property name="title">
         <string>Sensors</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTreeView" name="sensorList">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>period</tabstop>
  <tabstop>useCustomPeriod</tabstop>
  <tabstop>customPeriod</tabstop>
  <tabstop>summaryData</tabstop>
  <tabstop>allData</tabstop>
  <tabstop>sendNow</tabstop>
  <tabstop>sensorGroup</tabstop>
  <tabstop>sensorList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureReportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>717</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureReportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>717</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>period</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>customPeriod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
